src/precice/SolverInterface.hpp

```diff
@@ -559,33 +547,26 @@ class PRECICE_API SolverInterface {
   /**
    * @brief Checks if the data with given name is used by a solver and mesh.
    *
-   * @param[in] dataName the name of the data
-   * @param[in] meshID the id of the associated mesh
-   * @returns whether the mesh is used.
-   */
-  bool hasData(const std::string &dataName, int meshID) const;
-
-  /**
-   * @brief Returns the ID of the data associated with the given name and mesh.
-   *
-   * @param[in] dataName the name of the data
-   * @param[in] meshID the id of the associated mesh
-   *
-   * @returns the id of the corresponding data
+   * @param[in] meshName the name of the associated mesh
+   * @param[in] dataName the name of the data to check
+   * @returns whether the mesh contains the data.
    */
-  int getDataID(const std::string &dataName, int meshID) const;
+  bool hasData(
+      std::string_view meshName,
+      std::string_view dataName) const;
```

Comment:
This is not only changing the input arguments but an actual API command change. We have concluded on this right?

Reply:
Same reasoning as the C interface